tests/TestHelpers/DB.hs

```diff
@@ -36,8 +36,7 @@ runDB query = do
 createUser :: Text -> Example (Entity User)
 createUser ident = do
     insertEntity User
-        { userFirstName = "John" <> ident
-        , userLastName  = "Smith"
+        { userName      = "John " <> ident <> " Smith"
```

Comment:
It's not much different to do it in the parens way, and I don't really have a preference. I'll update it to use that style.